templates/base.html

```diff
@@ -8,7 +8,12 @@
     <title>{% block title %}AcousticBrainz{% endblock %}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" href="{{ url_for('static', filename='img/favicon.png') }}" type="image/x-icon" />
+
+    <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
```

Comment:
do we really need so many favicons? what are all of these used for? homscreen icons on an ipad? all my browser windows have 16x16 icons :)